### PR TITLE
Add Datadog options

### PR DESCRIPTION
**Pull request by Nhacsam**
I added the options accepted by the datadog api and an option to use the eu site as described here : #14

### CODE REVIEW

**Comment by ovhemert** [comment]
The additions that you made look great. Thanks  👍🏻

Would you mind updating the tests also to test the new options? You should be able to see failing test/missing coverage when you run `npm run test`.

**Comment by Nhacsam** [comment]
It should be ok now

File: test/datadog.test.js [hunk 1]
```diff
@@ -6,12 +6,6 @@ const sinon = require('sinon')
 
 const axios = require('axios')
 
-test('creates client', t => {
-  const client = new tested.Client({ apiKey: '1234567890' })
-  t.equals(client._apiKey, '1234567890')
```
Comment (Nhacsam) [inline test/datadog.test.js:11]: We are not supposed to access the `_apiKey` directly, right ?

if it is allowed, it is a regression.
It not, this case is now covered by the test "inserts sends com url and api key" where I check the apiKey is sended to the api.

**Comment by ovhemert** [comment]
LGTM,.. thanks!

**Pull request merged**